ts/src/profilers/heap-profiler-wrapper.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AllocationProfileNode} from '../v8-types';
+const profiler = require('bindings')('sampling_heap_profiler');
```

Comment:
Having "wrapper" in the file name is a bit ugly and not too informative (it does not communicate much on the purpose of the wrapping).

Could we have bindings.ts with approach like https://stackoverflow.com/a/44467330/848626 suggests?